src/kudu/consensus/leader_election.cc

```diff
@@ -507,28 +518,12 @@ FlexibleVoteCounter::IsPessimisticQuorumSatisfied() const {
   VLOG_WITH_PREFIX(3) << "Checking if pessimistic quorum is satisfied.";
 
   // Fetching all regions.
-  std::vector<std::string> regions;
+  std::set<std::string> regions;
```

Comment:
nit: unnecessary change. voter_distribution_ is a map. So vector insertion would have been unique.

Reply:
New IsMajoritySatisfiedInRegions() takes in a set. I'd actually change all call sites to take in a set as that makes more sense than a vector :)